Gemfile

```diff
@@ -5,4 +5,5 @@ gemspec
 
 group :test do
   gem 'rspec'
+  gem 'json_spec'
 end
```

Comment:
All dependencies for gem should live in the gemspec. You can add these in the gemspec as development dependencies.

http://guides.rubygems.org/patterns/#declaring-dependencies